src/pages/pupils/programmes/[programmeSlug]/units.tsx

```diff
@@ -107,10 +106,86 @@ export const getStaticProps: GetStaticProps<
         };
       }
 
+      curriculumData.sort((a, b) => {
```

Comment:
This could be moved to help function if there is ever another page that needs it.